.devcontainer/Dockerfile

```diff
@@ -12,22 +11,23 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get -y install --no-install-recommends apt-utils dialog && \
     apt-get -y install --no-install-recommends apt-transport-https ca-certificates software-properties-common libssl-dev libffi-dev \
         dotnet-sdk-3.1 build-essential gnupg-agent dnsutils httpie bash-completion curl wget git unzip jq lsb-release procps gnupg2 && \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9 && \
+    curl -O https://cdn.azul.com/zulu/bin/zulu-repo_1.0.0-2_all.deb && \
+    apt-get install ./zulu-repo_1.0.0-2_all.deb && \
+    apt-get update && \
+    apt-get -y install zulu11-jdk && \
```

Comment:
I'm wondering how long it takes to run the install.  If it is short enough then we can keep it as is.  If it takes a while then we might want to build an image to use.

Reply:
It's not any noticeable longer than before but we did push prebuilt image to dockerhub to `dockerhub retaildevcrew/ngsa-java-codespaces` with these installation

Reply:
Sounds good.  Let's go with this for the time being and if it becomes an issue then we'll revisit.